scripts.js

```diff
@@ -1,6 +1,11 @@
 function add_doggo() {
-    var src = "https://dog.ceo/api/breed/shiba/images";
-    show_image("https://dog.ceo/api/breed/shiba/images", 500,500, "Doggo");
+    fetch("https://dog.ceo/api/breed/shiba/images")
+        .then(function(response) {
+            return response.json();
+        })
+        .then(function(imageObj) {
```

Comment:
I like to name things like this "dogsArray" or something so it's obvious you're working with an array rather than just an object.